src/v3/src/types/schema.ts

```diff
@@ -42,7 +43,18 @@ export type FormBag = {
   dataSchema: DataSchemaBag;
 };
 
-export type IdxMessageWithName = IdxMessage & { name?: string; };
+export type WidgetMessage = Modify<IdxMessage, {
+  class?: string;
+  i18n?: {
+    key: string;
+    params?: unknown[];
+  };
+  message?: string | WidgetMessage[];
+  type?: 'list' | 'string';
```

Comment:
Do we still need `type`?

Reply:
Actually, I dont think so 🤔 

Let me try to get rid of it and see how it reacts.

Reply:
<3 yeah we should be able to rely on differentiating between the `message` union type as a replacement for checking `type`

Reply:
Works like a charm 🙏